R/course.R

```diff
@@ -0,0 +1,324 @@
+## see end of file for some cURL notes
+
+#' Download course materials
+#'
+#' Special-purpose function to download a folder of course materials. The only
+#' demand on the user is to confirm or specify where the new folder should be
+#' stored. Workflow:
+#' * User calls `use_course("SHORTLINK-GOES-HERE")`.
+#' * User is asked to notice and confirm the location of the new folder Specify
+#' `destdir` to skip this.
+#' * User is asked if they'd like to delete the ZIP file.
+#' * New folder is opened in the file manager, e.g. Finder or File Explorer.
+#'
+#' @param url Link to a ZIP file containing the materials, possibly behind a
+#'   shortlink. Function developed with DropBox and GitHub in mind, but should
+#'   work for ZIP files generally. See [download_zip()] for more.
+#' @param destdir The new folder is stored here. Defaults to working directory.
```

Comment:
Would it be better to default to the desktop? (Since everyone can find that)

Reply:
Good point!

Reply:
Done d06440d12084113d805f6c3494239b2a723c50d5. Usage examples at very top show this now.